### PR TITLE
make buildenv script work for Debian and derivatives

### PR DESCRIPTION
**Pull request by Be-ing**

### CODE REVIEW

**Comment by Be-ing** [comment]
Tested in a Debian 10.6 VM.

File: tools/debian_buildenv.sh [hunk 1]
```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
```
Comment (Holzhaus) [inline tools/debian_buildenv.sh:1]: Can you add a comment here that this script is also supposed to work with Ubuntu et. al.?

File: tools/debian_buildenv.sh [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+case "$COMMAND" in
+    name)
+        echo "No build environment needed for Ubuntu, please install dependencies using apt." >&2
+        ;;
+
+    setup)
+        source /etc/lsb-release 2>/dev/null
+        case "${DISTRIB_CODENAME}" in
+            bionic)
```
Comment (Holzhaus) [inline tools/debian_buildenv.sh:15]:
```suggestion
            bionic)  # Ubuntu 18.04 LTS (Bionic Beaver)
```

**Comment by Be-ing** [comment]
Merging this now to avoid merge conflicts for adding dput package...

**Pull request merged**